src/GO/Job.php

```diff
@@ -401,7 +402,8 @@ private function removeLockFile()
     /**
      * Execute a callable job.
      *
-     * @param  callable  $fn
+     * @param  callable $fn
```

Comment:
Can you please re-add the extra space? I'm using two spaces between the type and the var name everywhere else.